docs/source/tutorial02.rst

```diff
@@ -0,0 +1,194 @@
+
+    fom = LTIModel.from_matrices(A, B, C)
+
+We can get the internal representation of the |LTIModel| `fom`
+
+.. jupyter-execute::
+
+    fom
+
+From this, we see that the matrices were wrapped in |NumpyMatrixOperators|,
+while default values were chosen for :math:`D` and :math:`E` matrices
+(respectively, zero and identity). The operators in an |LTIModel| can be
+accessed directly, e.g., `fom.A`.
+
+We can also see some basic information from `fom`'s string representation
+
+.. jupyter-execute::
+
+    print(fom)
+
+To visualize the behavior of the `fom`, we can draw its magnitude plot
+
+.. jupyter-execute::
+
+    w = np.logspace(-2, 8, 50)
+    fom.mag_plot(w)
+    plt.grid()
+
+Plotting the Hankel singular values shows us how well an LTI system can be
+approximated by a reduced-order model
+
+.. jupyter-execute::
+
+    hsv = fom.hsv()
+    fig, ax = plt.subplots()
+    ax.semilogy(range(1, len(hsv) + 1), hsv, '.-')
+    ax.set_title('Hankel singular values')
+    ax.grid()
+
+As expected for a heat equation, the Hankel singular values decay rapidly.
+
+
+Running balanced truncation
+---------------------------
+
+First, we need the reductor object
+
+.. jupyter-execute::
+
+    bt = BTReductor(fom)
+
+Calling its :meth:`~pymor.reductors.bt.GenericBTReductor.reduce` method runs the
+balanced truncation algorithm. This reductor additionally has an `error_bounds`
+method which can compute the a priori :math:`\mathcal{H}_\infty` error bounds
+based on the Hankel singular values:
+
+.. jupyter-execute::
+
+    error_bounds = bt.error_bounds()
+    fig, ax = plt.subplots()
+    ax.semilogy(range(1, len(error_bounds) + 1), error_bounds, '.-')
+    ax.semilogy(range(1, len(hsv)), hsv[1:], '.-')
+    ax.set_xlabel('Reduced order')
+    ax.set_title(r'Upper and lower $\mathcal{H}_\infty$ error bounds')
+    ax.grid()
+
+To get a reduced-order model of order 10, we call the `reduce` method with the
+appropriate argument:
+
+.. jupyter-execute::
+
+    rom = bt.reduce(10)
+
+Instead, or in addition, a tolerance for the :math:`\mathcal{H}_\infty` error
+can be specified, as well as the projection algorithm (by default, the
+balancing-free square root method is used).
+
+We can compare the magnitude plots between the full-order and reduced-order
+models
+
+.. jupyter-execute::
+
+    fig, ax = plt.subplots()
+    fom.mag_plot(w, ax=ax, label='FOM')
+    rom.mag_plot(w, ax=ax, linestyle='--', label='ROM')
+    ax.legend()
+    ax.grid()
+
+and plot the magnitude plot of the error system
+
+.. jupyter-execute::
+
+    (fom - rom).mag_plot(w)
+    plt.grid()
+
+We can compute the relative errors in :math:`\mathcal{H}_\infty` or
+:math:`\mathcal{H}_2` (or Hankel) norm
+
+.. jupyter-execute::
+
+    print(f'Relative Hinf error: {(fom - rom).hinf_norm() / fom.hinf_norm():.3e}')
+    print(f'Relative H2 error:   {(fom - rom).h2_norm() / fom.h2_norm():.3e}')
+
+To compute the :math:`\mathcal{H}_\infty` norms, pyMOR uses the dense solver
+from Slycot.
```

Comment:
Explain what these means for other `Operators` than dense `NumpyMatrixOperators`?

Mention, that `V` and `W` can also be obtained from `reductor`?

Reply:
Done